recipes/dcmtk/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "DCMTK" %}
+{% set version = "3.6.7" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  url: https://dicom.offis.de/download/dcmtk/dcmtk{{ version | replace('.','') }}/dcmtk-{{ version }}.tar.gz
+  sha256: 7c58298e3e8d60232ee6fc8408cfadd14463cc11a3c4ca4c59af5988c7e9710a
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+    - ninja
+  host:
+    - openjpeg
+    - libpng
+    - libsndfile
+    - libtiff
+    - libxml2
+    - openssl
+    - zlib
+  run:
+    - libpng
+    - libsndfile
+    - libtiff
+    - openjpeg
+    - libxml2
+    - openssl
+    - zlib
+
+test:
+  commands:
+    - dcmdump --help
```

Comment:
Please add some existence tests for headers, cmake/pc files, libraries. For the libraries, define a jinja variable called SO_MAJOR and use it to check that the SO major version is as expected. e.g.

```
  - test -f $PREFIX/lib/libofstd.so.{{ SO_MAJOR }}  # [linux]
```

This will give advanced warning if there are any ABI breaking changes.

Reply:
Done in fd89a50d1f